spec/support/rails_template_with_data.rb

```diff
@@ -1,9 +1,16 @@
 apply File.expand_path("../rails_template.rb", __FILE__)
 
-%w{Post User Category}.each do |type|
+%w{Post User Category News}.each do |type|
```

Comment:
You're right, apparently the news model in the template isn't used and it is unnecessary.  I'll remove the last commit shortly.